pyxem/signals/diffraction_vectors2d.py

```diff
@@ -169,18 +169,6 @@ def get_unique_vectors(
         else:
             return unique_peaks
 
-    def get_magnitudes(self, out=None, rechunk=False):
```

Comment:
What sort of deletion is this? Was this functionality is a release version?

Reply:
This functionality is still there it's just handled by the `DiffractionVectors` class using the `map` function.  I had two functions at first but realized pretty quickly that was a problem when you might have n-dimenisional 2d vectors.

Reply:
okay, so calls to `.get_magnitudes` will still run then? If so great. Resolved.